README.md

```diff
@@ -101,7 +101,10 @@ The second step would be:
 
 ```swift
 let baseURL = NSURL(string: "https://myApi.com")!
-let reactor = Reactor<Author>(persistencePath: path, baseURL:baseURL)
+let configuration = FlowConfiguration(usingPersistence: .True("path_to_persistence"))
```

Comment:
Should be `.Yes("path_to_persistence")` since it's a `ShouldUsePersistence` not a `Bool`